src/components/form.tsx

```diff
@@ -193,6 +194,20 @@ const HHLangField = ({ handleChangeValues, changeValues }: FormProps) => {
           >
             {HOUSEHOLD_LANGUAGES.MALAY.DISPLAY}
           </ToggleButton>
+          <ToggleButton
+            id="type-cb-5"
+            variant="outline-primary"
+            value={HOUSEHOLD_LANGUAGES.TAGALOG.CODE}
+          >
+            {HOUSEHOLD_LANGUAGES.TAGALOG.DISPLAY}
+          </ToggleButton>
+          <ToggleButton
```

Comment:
This space will get more tighter as more languages are put in here. Maybe a multi-select dropdown would help here.

Reply:
Good for future development